src/cli/platform/sgx/crl.rs

```diff
@@ -22,10 +31,43 @@ pub struct CrlCache {}
 
 impl CrlCache {
     pub fn execute(self) -> anyhow::Result<ExitCode> {
-        let dir = sgx_cache_dir()?;
-        save_file(CERT_CRL, &dir)?;
-        save_file(PROCESSOR_CRL, &dir)?;
-        save_file(PLATFORM_CRL, &dir)?;
+        let mut dest_file = sgx_cache_dir()?;
+        dest_file.push("crls.der");
+        let dest_file = Path::new(&dest_file);
```

Comment:
Oh, missed this..  why `Path::new`???

Reply:
Borrower issues, couldn't display the error message with the path otherwise

Reply:
Figured it out, fixed.